website/docs/docs/develop/develop-in-the-cloud.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Develop in the Cloud"
+id: develop-in-the-cloud
+description: "Develop, test, run, and build in the Cloud IDE."
+sidebar_label: "Develop in the Cloud"
+---
+
+The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly.
+
+The IDE leverages the open-source [dbt-rpc](/docs/reference/commands/rpc) plugin to recompile only the changes made in your project.
+
+## Prerequisites
+
+To develop in the Cloud IDE, make sure you have the following:
+
+- Your dbt project must be compatible with dbt version 0.15.0 or higher. The dbt IDE is powered by the [dbt-rpc](/docs/reference/commands/rpc) which was overhauled in dbt v0.15.0
+- You must have a [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/dbt-cloud/access-control/cloud-seats-and-users)
+- You must have a git repository set up and your git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application) and [Importing a project by git URL](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url) for detailed setup instructions
+- Your dbt project must be connected to a [data platform](/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database)
+- You must have a [**development environment** and **development credentials**](/docs/develop/develop-in-the-cloud#set-up-and-access-the-cloud-ide) set up
+- The environment must be on dbt version 1.0 or higher
+
+
+## Set up and access the Cloud IDE
+
+:::info📌 
+
+New to dbt? Check out our [Getting Started guide](/docs/guides/getting-started) to build your first dbt project in the Cloud IDE!
+
+:::
+
+In order to start experiencing the great features of the Cloud IDE, you need to first set up your **Development environment** and **Development credentials.** 
+
+If you’re new to dbt, you will automatically add this during the project setup. However, if you have an existing dbt Cloud account, you may need to create a development environment and credentials manually to use the Cloud IDE. 
```

Comment:
> If you’re new to dbt, you will automatically add this during the project setup. 

awkward: the "this" in the sentence is referring to something that's not in the same paragraph. 

quick fix: move sentence to previous paragraph to connect those ideas together